### DISCUSSION TITLE
Add Episode Renaming options like HDR

### DISCUSSION THREAD

**Discussion by Rouzax** [pymedusa/Medusa#7515] [Ideas & Feature requests]
Since more and more episodes are being released with HDR I would like the option to include HDR as tag if it is present in the episode.
Right now I use `Season %0S/%SN - S%0SE%0E - [%SQN][%RG]` which gives me: `The Witcher - S01E01 - [1080p WEB x264][AMRAP].mkv`

It would be nice to have more control like for instance Radarr where I have: `{Movie Title} ({Release Year}) - {[Edition Tags]}{[Quality Full]}[{MEDIAINFO VIDEOCODEC}{ MediaInfo HDR}][{MEDIAINFO AUDIOCODEC}{ MEDIAINFO AUDIOCHANNELS]}{MediaInfo AudioLanguagesAll}{[Release Group]}` which gives me: `Jumanji (1995) - [Remux-2160p][H265 HDR][TRUEHD ATMOS 7.1][EN][EPSiLON].mkv`

**Comment by p0psicles** [comment]
We could definitely enhance the renaming. I also like the radarr concept. But we could use guessit and optionally knowit sources

**Comment by Rouzax** [comment]
Would it be possible to pick this up?
Currently, it is hard to see in a show overview which episodes were downloaded with HDR and which are not.
I know a few episodes got downloaded with HDR but I need to check the torrents now to identify
![image](https://user-images.githubusercontent.com/4103090/101290092-7b9d6480-3800-11eb-92f8-944147a45b50.png)

**Comment by p0psicles** [comment]
Don't think we currently have the time for this. So it would need to get added through external pr

**Comment by Rouzax** [comment]
I understand, it was worth a shot.
